Cargo.toml

```diff
@@ -53,3 +53,7 @@ members = [
     "otl-normalizer",
     "fontc_crater",
 ]
+
+[patch.crates-io]
```

Comment:
I think we should view this as submit-blocking, lets just release fontations.

Reply:
totally, just waiting for a review.